code/modules/hydroponics/plants_herb.dm

```diff
@@ -150,14 +160,15 @@
 	force_seed_on_harvest = 1
 	special_proc = 1 // for tuna plant
 	harvested_proc = 1 // for tuna plant
-	isgrass = 0 // for some reason, isgrass forces you not to gave a G3 sprite and stops a bunch of the "fully grown plant" shit from happening :[
+	isgrass = 0 // for some reason, isgrass forces you not to gave a G3 sprite and stops a bunch of the "fully grown plant" shit from happening :[ //TODO: REVIEW IF THIS IS STILL RELEVANT WITH G4s!!!
```

Comment:
Has this been looked at?

Reply:
I'll have to do some work to test this but I'll look into it!

Reply:
Seems like everything's fine now :) I must have fixed it without realizing